ui/src/api/index.ts

```diff
@@ -35,6 +35,7 @@ export interface Api {
   setRating(resourceId: number, token: string, rating: number): Promise<void | null>;
   getRefreshToken(refreshToken: string): Promise<ITokenInfo>;
   getAccessToken(accessToken: string): Promise<ITokenInfo>;
+  info(token: string): Promise<IUserInfo>;
```

Comment:
I guess we can rename this to userInfo. WDYT?

Reply:
we have `user` as AuthStore instnace in the root store so here I have used `info`. if I rename it to `userInfo` then `user.userInfo` might be a bit confusing , WDYT?

Reply:
Can we rename it to `profile` which will call the function as `user.Profile`